CHAOSScon/2020EU/desc.md

```diff
@@ -12,6 +12,22 @@ Co-located with FOSDEM
 
 January 31, 2020
 
+## Registration
+
+Registration will open soon!
+
+### Diversity and Inclusion
+
+To encourage and increase the participation of a diverse community, we are pleased to offer waivers for registration based on diversity and need. To get a your registration waived, please contact [Georg Link](linkgeorg@gmail.com)
+
+All speakers and attendees are required to adhere to our [Event Code of Conduct](https://events.linuxfoundation.org/about/code-of-conduct/).
+
+If you have any concerns regarding code of conduct issues prior to the event or during the event, please contact [Dawn Foster](dawn@dawnfoster.com) or [Georg Link](linkgeorg@gmail.com)
```

Comment:
The code of conduct has a separate email address (chaoss-inclusion@lists.linuxfoundation.org), how about we use that for CoC issues?

Reply:
See below

Reply:
I think there are two concerns here. One is the general code of conduct - reporting harassers and such. 

The other is the CoC for the event itself - Dawn suggested we use the LFs event CoC for that and I agree. For the event CoC, I think it is important to have an on-site contact and a point of contact where issues may be handled in advance (i.e. "if this person is there, we need to be seperated", ... "or I require an escort", or "I have ____ issue specific to the venue and need support or guidance".)

Reply:
okay, I'm happy to be a contact. Does the link need to include "mailto:"? Like this:

    [Georg Link](mailto:linkgeorg@gmail.com)